### ISSUE TITLE
can't deal with subdirectories and multible repositories 

### ISSUE THREAD

**Issue by RicoP** [abel0b/setup-premake#5]
This is my action script: 

```yaml
name: Windows Build

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

permissions:
  contents: read

jobs:
  build:
    runs-on: windows-latest
    
    steps:
    # checkout rose.parser with it's dependencies.
    - uses: actions/checkout@v3
      with:
        repository: RicoP/repo1
        path: ./repo1
    
    - uses: actions/checkout@v3
      with:
        repository: RicoP/repo2
        path: ./repo2

    - name: Setup premake
      uses: abel0b/setup-premake@v2
      with:
        version: "5.0.0-beta1"


    - name: Show Working directory
      working-directory: ${{env.GITHUB_WORKSPACE}}
      run: DIR
     #Mode                 LastWriteTime         Length Name
     #----                 -------------         ------ ----
     #d----           4/14/2022  3:53 PM                .premake
     #d----           4/14/2022  3:53 PM                repo1
     #d----           4/14/2022  3:53 PM                repo2

# this works
    - name: test
      if: always()
      run: premake5 --help

# this works
    - name: test
      if: always()
      working-directory: repo1
      run: ../.premake/premake5 --help

# this DOESN'T work
    - name: test
      if: always()
      working-directory: repo1
      run: premake5 --help


```

It seems like when having subdirectories the shell can't find premake5 in the path

**Comment by abel0b** [comment]
Fixed in setup-premake@v2.1

**Comment by RicoP** [comment]
thanks~
